app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/sdk/model/Fido2CredentialStoreImpl.kt

```diff
@@ -0,0 +1,89 @@
+package com.x8bit.bitwarden.data.vault.datasource.sdk.model
+
+import com.bitwarden.fido.Fido2CredentialAutofillView
+import com.bitwarden.sdk.Fido2CredentialStore
+import com.bitwarden.vault.Cipher
+import com.bitwarden.vault.CipherView
+import com.x8bit.bitwarden.data.auth.repository.AuthRepository
+import com.x8bit.bitwarden.data.autofill.util.isActiveWithFido2Credentials
+import com.x8bit.bitwarden.data.platform.annotation.OmitFromCoverage
+import com.x8bit.bitwarden.data.vault.datasource.sdk.VaultSdkSource
+import com.x8bit.bitwarden.data.vault.repository.VaultRepository
+
+/**
+ * Primary implementation of [Fido2CredentialStore].
+ */
+@OmitFromCoverage
+class Fido2CredentialStoreImpl(
+    private val vaultSdkSource: VaultSdkSource,
+    private val authRepository: AuthRepository,
+    private val vaultRepository: VaultRepository,
+) : Fido2CredentialStore {
+    override suspend fun allCredentials(): List<CipherView> {
+        vaultRepository.sync()
+        return vaultRepository.ciphersStateFlow.value.data.orEmpty()
```

Comment:
⚠️ This should only be returning active ciphers that have Fido2 credentials; basically the same as in line 32...34.